hardware_interface/include/hardware_interface/components/base_interface.hpp

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 ROS2-Control Development Team
```

Comment:
```suggestion
// Copyright 2020 ros2_control Development Team
```
Please use everywhere, except where autocorrect doesn't allow us (e.g ROS discourse...)